astropy/cosmology/core.py

```diff
@@ -80,6 +80,13 @@
 kB_evK = const.k_B.to(u.eV / u.K)
 
 
+u.def_unit(['redshift'],  prefixes=False, namespace=globals(),
```

Comment:
I'm not sure if this is the place to define the unit. Perhaps in ``/equivalencies.py``?